com.unity.ml-agents/Tests/Editor/Communicator/RpcCommunicatorTests.cs

```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace MLAgents.Tests.Communicator
+{
+    [TestFixture]
+    public class RpcCommunicatorTests
+    {
+        [Test]
+        public void TestCheckCommunicationVersionsAreCompatible()
+        {
+            var unityVerStr = "1.0.0";
+            var pythonVerStr = "1.0.0";
```

Comment:
```suggestion
            var pythonVerStr = "1.1.0";
```
Just to make it slightly interesting